services/openshift/scripts/openshift_provision

```diff
@@ -21,9 +21,9 @@ wait_for_openshift_api() {
       ((ATTEMPT++))
     done
 
-    # Wait for API to be accessible
+    # Use health URL to check OpenShift readiness
     ATTEMPT=0
-    until curl -ksSf https://127.0.0.1:8443/api > /dev/null 2>&1 || [ $ATTEMPT -eq 30 ]; do
+    until curl -ksSf https://127.0.0.1:8443/healthz/ready > /dev/null 2>&1 || [ $ATTEMPT -eq 30 ]; do
```

Comment:
On a different note, should we increase the timout? 30s is maybe a bit low? I think we had cased were some people where hitting the timeout even though everything was still deploying!? Maybe a minute?